### DISCUSSION TITLE
Documentation on the GPO settings

### DISCUSSION THREAD

**Discussion by Krazy-Kiwi** [Open-Shell/Open-Shell-Menu#1332] [General]
Is there any documentation, examples, or advice on the GPO settings to enable Open-Shell within a domain environment?

In this case, we are using Open-Shell within a school environment with a tight lockdown policy only to supply what the school needs students to have and restrict access.

The main issue we need to solve is the first time the user signs in, Open-Shell always opens the menu. After this, it opens correctly with the locked-down configuration.

**Comment by ge0rdi** [comment]
> Is there any documentation, examples, or advice on the GPO settings to enable Open-Shell within a domain environment?

You can check `Administrative Settings` section in `Open-Shell` help.

Also `PolicyDefinitions.zip` (located in `Open-Shell` install directory) contains `PolicyDefinitions.rtf` document with some instructions.

> The main issue we need to solve is the first time the user signs in, Open-Shell always opens the menu. After this, it opens correctly with the locked-down configuration.

To avoid showing Settings dialog for the first time, you need to _enable_ `Menu Style` policy setting (it is enough to set default style, no need to lock it to specific value).

**Reply by Krazy-Kiwi** [reply]
Thank you, the Menu Style worked a treat. I'm slowly going through the Policy document and this will be a later stage of the project as the school goes live this week.
